nix/onchain.nix

```diff
@@ -54,6 +54,22 @@ in
             };
           };
 
+          fourmolu = types.submodule {
+            options = {
+              package = lib.mkPackageOption pkgs "fourmolu" {
+                default = [ "haskell" "packages" "ghc924" "fourmolu_0_9_0_0" ];
+              };
+            };
+          };
+
+          applyRefact = types.submodule {
+            options = {
+              package = lib.mkPackageOption pkgs "apply-refact" {
+                default = [ "haskell" "packages" "ghc924" "apply-refact_0_10_0_0" ];
```

Comment:
Could you test if *ghc925* works?

Reply:
What works in place of GHC and tool version depends on version of `nixpkgs` used. If `nixpkgs` is new enough (if you're reading this in the future: old enough) and has `ghc925` attribute it'll work (I use it on my project and works fine)